testing/test.ts

```diff
@@ -1,255 +1,10 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
-import { test, runIfMain } from "./mod.ts";
-import {
-  assert,
-  assertEquals,
-  assertStrictEq,
-  assertThrows,
-  assertThrowsAsync
-} from "../testing/asserts.ts";
+import { runIfMain } from "./mod.ts";
+
 import "./format_test.ts";
 import "./diff_test.ts";
 import "./pretty_test.ts";
 import "./asserts_test.ts";
 import "./bench_test.ts";
 
-test(function testingAssertEqualActualUncoercable() {
```

Comment:
These do need to be moved, but can you do this in a separate PR?

Reply:
Yes. I'll drop this commit.